.github/CODEOWNERS

```
@@ -0,0 +1,2 @@
+* @castrojo
+* @imbev
```

```suggestion
* @castrojo @imbev
```
These should be on the same line